tests/cgas_overflow.rs

```diff
@@ -0,0 +1,96 @@
+use fuel_vm::consts::*;
```

Comment:
This file was moved to `tests/cgas.rs` and should be removed.

Reply:
https://github.com/FuelLabs/fuel-vm/pull/173/commits/62af3eeb75ef71b79f1f61e21341690021a16d6d